translator/util/sdkutil.go

```diff
@@ -97,19 +99,25 @@ func defaultECSRegion() string {
 	return ecsutil.GetECSUtilSingleton().Region
 }
 
-func detectRegion(mode string, credsConfig map[string]string) (region string) {
+func detectRegion(mode string, credsConfig map[string]string) (region string, regionType string) {
 	region = SDKRegionWithCredsMap(mode, credsConfig)
+	regionType = agentinfo.REGION_NOT_FOUND
```

Comment:
I think if this happens, the agent will shut down before being able to reach out to the backend to send this metric. I will leave this in any way.